_packages/native-preview/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+    "compilerOptions": {
+        "module": "node16",
+        "strict": true,
+        "forceConsistentCasingInFileNames": true,
```

Comment:
This is the default now [right?](https://github.com/microsoft/TypeScript/issues/17542)

Reply:
This is still just a copy of the config from another dir, but I can improve all of this if you want.